packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -54,6 +55,76 @@ type TabsTypeProps =
       type: 'pill';
     };
 
+type TabsOverflowProps =
+  | {
+      /**
+       * If specified, this prop will be used to show a custom button when overflow happens,
+       * i.e. when there is not enough space to fit all the tabs.
+       *
+       * Expects a function that takes the number of items that are visible
+       * and returns the `ReactNode` to render.
+       *
+       * Not applicable to `type: 'pill'` or `orientation: 'vertical'`.
+       *
+       * @example <caption>Uses the overflow button to add a dropdown that contains hidden tabs</caption>
+       *  const [index, setIndex] = React.useState(0);
+       *  return (
+       *    <Tabs
+       *      overflowButton={(visibleCount) => (
+       *        <DropdownMenu
+       *          menuItems={(close) =>
+       *            Array(items.length - visibleCount)
+       *              .fill(null)
+       *              .map((_, _index) => {
+       *                const index = visibleCount + _index + 1;
+       *                const onClick = () => {
+       *                  // click on "index" tab
+       *                  setIndex(index - 1);
+       *                  close();
+       *                };
+       *                return (
+       *                  <MenuItem key={index} onClick={onClick}>
+       *                    Item {index}
+       *                  </MenuItem>
+       *                );
+       *              })
+       *          }
+       *        >
+       *          <IconButton style={{ paddingTop: '12px', margin: '4px', height: 'auto' }} styleType='borderless'>
+       *            <SvgMoreSmall />
+       *          </IconButton>
+       *        </DropdownMenu>
+       *      )}
+       *      onTabSelected={setIndex}
+       *      activeIndex={index}
+       *    >
+       *      {tabs}
+       *    </Tabs>
+       *  );
+       */
+      overflowButton?: (visibleCount: number) => React.ReactNode;
+      /**
+       * Orientation of the tabs.
+       * @default 'horizontal'
+       */
+      orientation?: 'horizontal';
+      /**
+       * Type of the tabs.
+       *
+       * If `orientation = 'vertical'`, `pill` is not applicable.
+       * @default 'default'
+       */
+      type?: 'default' | 'borderless';
+    }
+  | {
+      orientation: 'vertical';
+      type?: 'default' | 'borderless';
+    }
+  | {
+      type: 'pill';
+      orientation?: 'horizontal';
+    };
```

Comment:
This isn't quite right. I get errors on the 'PillTabs' and 'Vertical' stories:
![image](https://user-images.githubusercontent.com/37936169/217636782-46ad2195-7acf-46c3-aea1-7d891e51cb08.png)
Does anyone know what the issue is? @gretanausedaite @mayank99 @adamhock

Reply:
Do you get the same error in test app? I think this can be just in storybook (similar errors in combobox).

Reply:
I don't get the same errors, and yeah, I see the errors in ComboBox. Thanks Greta!